code/drasil-docLang/Drasil/DocLang.hs

```diff
@@ -12,13 +12,14 @@ module Drasil.DocLang (
   RefSec(..), RefTab(..), StkhldrSec(..), StkhldrSub(Client, Cstmr),
   TConvention(..), TraceabilitySec(TraceabilityProg), TSIntro(..), TUIntro(..),
   -- DocumentLanguage.Definitions
-  Field(..), Fields, InclUnits(IncludeUnits), Verbosity(Verbose), ddefn,
+  Field(..), Fields, InclUnits(IncludeUnits), Verbosity(..), ddefn,
```

Comment:
Note that, in general, we prefer to be precise in our imports, `Verbose` is better than `..`. I won't block things just for that, but try to keep precise imports in the future.